cpp/src/parquet/statistics.h

```diff
@@ -259,6 +264,14 @@ class PARQUET_EXPORT Statistics {
   /// \brief Plain-encoded maximum value
   virtual std::string EncodeMax() const = 0;
 
+  /// \brief Return the minimum value exact flag if set.
+  /// It will be true if there was no truncation.
+  virtual std::optional<bool> is_min_value_exact() const = 0;
```

Comment:
Should we consider simplifying it as `virtual bool is_min_value_exact() const = 0;` which returns false if the value is missing internally?